presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java

```diff
@@ -94,8 +101,26 @@ private static HiveTableDefinition allHiveSimpleTypesParquetTableDefinition()
                         "   c_boolean            BOOLEAN," +
                         "   c_binary             BINARY" +
                         ") " +
-                        "STORED AS PARQUET")
-                .setDataSource(dataSource)
-                .build();
+                        "STORED AS PARQUET");
+    }
+
+    private static HiveDataSource getTextFileDataSource()
+    {
+        return createResourceDataSource(format(tableNameFormat, "textfile"), String.valueOf(ThreadLocalRandom.current().nextLong(Long.MAX_VALUE)), "com/facebook/presto/tests/hive/data/all_types/data.textfile");
```

Comment:
In next commit (0c07572e83e33b4ba1757a25a8aded5b1b37750b) this method is used only once.
I simply extracted refactor (creating those repetitions) to make the actual commit with logical change more succinct and easier to follow.
If you don't like it maybe the best would be to squash 2866c60cd7d2676e123b6fa0b614c9fece0aec72 onto 0c07572e83e33b4ba1757a25a8aded5b1b37750b?